core-ui/src/components/App/AppContext.js

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+export const AppContext = createContext({
```

Comment:
wouldn't it be cleaner to create a `SchemaContext`?
So we don't end up with a single monolithic context for everything?

Reply:
When Luigi is removed, we'll add its microcontext replacement here, no point to divide it to multiple contexts

Reply:
I guess we can wait until then with this topic